cmd/notification.go

```diff
@@ -231,11 +231,13 @@ func (sys *NotificationSys) AddRemoteTarget(bucketName string, target event.Targ
 	if targetMap == nil {
 		targetMap = make(map[event.TargetID]event.RulesMap)
 	}
-	targetMap[target.ID()] = rulesMap.Clone()
+	rulesMap = rulesMap.Clone()
+	targetMap[target.ID()] = rulesMap
 	sys.bucketRemoteTargetRulesMap[bucketName] = targetMap
+	rulesMap.Add(sys.bucketRulesMap[bucketName])
```

Comment:
you would need clone `rulesMap` again, else `targetMap[target.ID()]` will have updated `rulesMap`